framework/captcha/Captcha.php

```diff
@@ -91,7 +92,7 @@ public function init()
     {
         parent::init();
 
-        static::checkRequirements();
+        $this->checkRequirements();
```

Comment:
In the light of other changes `checkRequirements` shold be removed from `Captcha` widget and kept only at `CaptchaAction` as it handles the image rendering.